### PR TITLE
This pull request allows to enable testing mariadb-container into CVP pipeline

### PR DESCRIPTION
**Pull request by phracek**
* container-common-scripts are updated
* Couple fixes in test-lib-mysql.sh and run-remote-openshift script
* add examples and test-openshift.yaml symlinks to test directory

### CODE REVIEW

**Comment by phracek** [comment]
[test-all]

**Comment by phracek** [comment]
[test-all]

File: test/test-lib-mysql.sh [hunk 1]
```diff
@@ -129,22 +129,14 @@ function test_mariadb_integration() {
...
-  # Try pulling the image first to see if it is accessible
-  PUBLIC_IMAGE_NAME=${PUBLIC_IMAGE_NAME:-$(ct_get_public_image_name "${OS}" "${BASE_IMAGE_NAME}" "${VERSION}")}
-  if docker pull "${PUBLIC_IMAGE_NAME}"; then
-    ct_os_test_image_stream_template "${THISDIR}/../imagestreams/mariadb-${OS%[0-9]*}.json" "${THISDIR}/../examples/mariadb-ephemeral-template.json" mariadb "-p MARIADB_VERSION=${VERSION}${tag}"
-  else
-    echo "Warning: ${PUBLIC_IMAGE_NAME} could not be downloaded via 'docker'"
-    # ignore possible failure of this test for centos images
-    [ "${OS}" == "rhel7" ] && false "ERROR: Failed to pull image"
-  fi
+  ct_os_test_image_stream_template "${THISDIR}/imagestreams/mariadb-${OS%[0-9]*}.json" "${THISDIR}/examples/mariadb-ephemeral-template.json" mariadb "-p MARIADB_VERSION=${VERSION}${tag}"
```
[the remark this answers is not in the document]
Comment (phracek) [reply test/test-lib-mysql.sh:139]: We do not need to pull images into the host OS. We need to import the image into OpenShift and this is done by testing `ct_os_test_image_stream_template` which imports the image by itself. Like it is specified in JSON file in the imagestream directory

[the remark this answers is not in the document]
Comment (zmiklank) [reply test/test-lib-mysql.sh:139]: Thank you for the explanation. I would also suggest to add this to the commit message, if you find it appropriate.

**Review by zmiklank — APPROVED** [review]
See my comment. LGTM, thanks.

**Comment by phracek** [comment]
[test-all]

**Pull request merged**